x-pack/plugins/observability_solution/inventory/public/components/entities_grid/index.tsx

```diff
@@ -61,8 +194,49 @@ export function EntitiesGrid() {
       )}
       columns={columns}
       columnVisibility={{ visibleColumns, setVisibleColumns }}
-      rowCount={value.entities.length}
-      renderCellValue={CellValue}
+      rowCount={entities.length}
```

Comment:
nit: You can use the following props to make the grid look less like excel cells 🐒 

```

 gridStyle={{
        border: 'horizontal',
        header: 'underline',
  }}
```